src/Legacy/Storage.php

```diff
@@ -717,6 +717,12 @@ private function searchSingleContentType($query, $contenttype, $fields, array $f
                     // Build the LIKE, lowering the searched field to cover case-sensitive database systems
                     $fieldsWhere[] = sprintf('LOWER(%s.%s) LIKE LOWER(%s)', $table, $field, $this->app['db']->quote('%' . $word . '%'));
                 }
+            } elseif ($fieldconfig['type'] === 'repeater') {
+                foreach ($query['words'] as $word) {
+                    // Build the LIKE, lowering the searched field to cover case-sensitive database systems
+                    $word = '%'.$word.'%';
+                    $fieldsWhere[] = "((LOWER(bolt_field_value.value_text) LIKE LOWER('$word')) OR (LOWER(bolt_field_value.value_string) LIKE LOWER('$word')))";
+                }
```

Comment:
The `Bolt\Legacy` classes have been closed to additions since 2.2.

Reply:
But are currently in use for search, or i missed something?